packages/node_modules/samples/browser-single-party-call/test/wdio/spec/dial-and-reject.js

```diff
@@ -0,0 +1,50 @@
+import testUsers from '@ciscospark/test-helper-test-users';
+import setValue from '../lib/set-value';
+
+describe(`samples/browser-single-party-call`, () => {
+  describe(`dial and reject`, () => {
+    let mccoy, spock;
+
+    const browserSpock = browser.select(`browserSpock`);
+    const browserMccoy = browser.select(`browserMccoy`);
+
+    before(`create test users`, () => testUsers.create({count: 2})
+      .then((users) => {
+        spock = users[0];
+        mccoy = users[1];
+      }));
+
+    step(`loads the app`, () => {
+      browser.url(`/packages/node_modules/samples/browser-single-party-call?usefakemedia`);
+    });
+
+    step(`connects mccoy's browser`, () => {
+      browserMccoy.waitUntil(() => browserMccoy.getTitle() === `Sample: Single Party Calling`);
+      browserMccoy.execute(setValue, `[placeholder="Your access token"]`, mccoy.token.access_token);
+      browserMccoy.click(`[title="connect"]`);
```

Comment:
We've been creating test helper elements objects that contain all these locator strings in the widgets. So you would import it then use `elements.connectButton`. Almost like a page-objects lite.

Reply:
historically, i've been against that strategy because it mostly just gives you even more files to look at to understand a test. that said, i accept that it makes sense when you're writing a large number of tests to interact with a small number of elements. Given the narrow scope of these tests, do you think it adds much of any thing?

Reply:
I only brought it up because I saw the selectors repeated across the different test scenarios. It doesn't really make sense for a one time use element selector. I do think these tests would benefit from it by reducing the amount of "magic strings" throughout.

Reply:
fair enough. I'd say, let's leave it as-is for now, and maybe once we've got a few more samples built, we can standardize them all a bit better than they're looking right now and then add those helpers.

Reply:
Sure thing